system-tests/lib/system-tests.ts

```diff
@@ -960,11 +972,17 @@ const systemTests = {
         } catch (err) {
           // firefox has issues with recording video. for now, ignore snapshot diffs that only differ in this error.
           // @see https://github.com/cypress-io/cypress/pull/16731
-          if (!(options.browser === 'firefox' && isVideoSnapshotError(err))) {
+          if (!(options.browser === 'firefox' && (isVideoSnapshotError(err) || isFirefoxConnectionError(err)))) {
             throw err
           }
 
-          console.log('(system tests warning) Firefox failed to process the video, but this is being ignored due to known issues with video processing in Firefox.')
+          if (isVideoSnapshotError(err)) {
+            console.log('(system tests warning) Firefox failed to process the video, but this is being ignored due to known issues with video processing in Firefox.')
+          }
+
+          if (isFirefoxConnectionError(err)) {
+            console.log('(system tests warning) Firefox failed to connect initially, but this is being ignored due to known issues with Firefox taking longer than normal to launch.')
```

Comment:
Are there issues logged around this that we can add a link to? I assume our consumers see this error as well. Would it be reasonable to increase the initial connection timeout for firefox instead of hiding the error in our tests?

Reply:
It's not an error, just a warning that appears when the browser is taking "longer than normal" to connect, to act as reassurance that Cypress is still trying to connect. An actual timeout will still error the tests out. I think most of the time users don't see this.

Maybe the warning should be removed altogether, there are issues created by users who are confused what this output means.

Reply:
Gottchya, we could probably change the verbiage to something like:

`still waiting to connect to firefox browser` instead to logging `failed to connect`.